website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
The first `save` needs to occur before contributors are added, but that could be fixed with additional cases and more better error handling. The second is to save the changes made since the first.